docs/3.0.0-beta.x/community/contribute-with-docs.md

```diff
@@ -0,0 +1,486 @@
+
+### Submitting an Externally Hosted Tutorial
+
+If you have written a **Tutorial**, **How-to** or other teaching material that is _outside_ of the documentation. Then you may submit a pull request with a link to your guide on the [Articles](/3.0.0-beta.x/articles/) page.
+
+### Docs Style Guide
+
+This style guide exists to show and explain the standards and style of how our documentation is written. We are offering this style guide to help you stay consistent with the existing documentation.
+
+---
+
+#### Favor _Yarn_ commands over _npm_ commands
+
+We recommend that users develop using the Yarn package manager. Therefore, the documentation shows commands first using `yarn` and then showing the equivalent `npm` command. Please consider the below examples:
+
+**Example 1**
+
+- Install Strapi with the following commands:
+
+Using **Yarn**:
+
+```
+yarn create strapi-app my-project --quickstart
+```
+
+Using **npm/npx**:
+
+```
+npx create-strapi-app my-project --quickstart
+```
+
+**Example 2**
+
+- You can access the Strapi Help instructions from your command line with:
+
+Using **Yarn**:
+
+```
+yarn strapi
+```
+
+Using **npm**:
+
+```
+npm run strapi help
+```
+
+---
+
+#### When giving instructions, say, _"you"_ not _"we"_
+
+Clear and concise instructions and language can help individuals to get the full advantages from using the Docs. Whenever an action needs doing, you use the word, "you" rather than "we." To clarify this, consider the following examples:
+
+**Example 1**
+
+1. **"You will next open your Chrome browser. Then, open a tab."**
+2. **"We will next open our Chrome browser. Then, we will open a tab."**
+
+In the above, the #1 phrase is precise, and the reader knows what is expected. In the #2 phrase, the additional words "we will" is needed to keep the sentence relatively clear. **Note:** "Then, we open a tab." - is awkward and unclear.
+
+**Example 2**
+
+1. **"Download Node.js and install it onto your computer."**
+2. **"We download Node.js, and then we install it onto our computers."**
+
+In the above, the #1 phrase is precise. #2 is actually confusing.
+
+Therefore, when you write instructions, use "you." This guide helps to keep the documentation clearly written.
+
+---
+
+#### Ensure to use trade, project and service provider names correctly
+
+Strapi is back-end and front-end agnostic. Many different front-ends and databases can (and will) be used. Also, to front-ends and back-ends, many packages, tools, and third-party service providers are often used with Strapi. Referring to the projects and services correctly is essential. If you use incorrect terms to refer to external services, users could get confused when further research outside the Strapi docs is required.
+
+Common projects and how to refer to them:
+
+| Incorrect Usage     | Correct usage |                   Example                   |
```

Comment:
Maybe this table should go in a specific file so you can have a dedicated section in the docs?

Reply:
I thought about this. But there are several reasons why its impractical at the moment. 
1. Creating smaller pages that suite the left-hand navigation is basically impossible with VuePres
2. This particular section is still very small
3. This entire doc was written in a modular way and I intend for it to grow as we discover new styles, new terms (in this case) and more examples etc..  Eventually, this would be it's own page probably.

For now this is the best solution.